project/LibDependencies.scala

```diff
@@ -15,8 +15,8 @@ object LibDependencies {
   }
 
   def compileDependencies(playVersion: PlayVersion) = playVersion match {
-    case PlayVersion.Play28 => "com.typesafe.play" %% "play" % "2.8.21"
-    case PlayVersion.Play29 => "com.typesafe.play" %% "play" % "2.9.0"
+    //case PlayVersion.Play28 => "com.typesafe.play" %% "play" % "2.8.21"
```

Comment:
Delete

Reply:
I will delete this commented lines as well.Thanks.